charts/opentelemetry-demo/values.yaml

```diff
@@ -583,7 +583,7 @@ components:
     enabled: true
     useDefault:
       env: true
-
+    serviceType: ClusterIP
```

Comment:
This was a minor cleanup, I think it was missing accidentally.

Reply:
we have another PR to remove this from being required on all components (it will default to ClusterIP).

Reply:
ok, I'll prune it then.